model/network/registry.yaml

```diff
@@ -236,3 +236,46 @@ groups:
         type: string
         brief: 'The network interface name.'
         examples: [ 'lo', 'eth0' ]
+      - id: network.connection.state
+        type:
+          members:
+            - id: close
+              value: 'close'
+              stability: experimental
+            - id: close_wait
+              value: 'close_wait'
+              stability: experimental
+            - id: closing
+              value: 'closing'
+              stability: experimental
+            - id: delete
+              value: 'delete'
+              stability: experimental
+            - id: established
+              value: 'established'
+              stability: experimental
+            - id: fin_wait_1
+              value: 'fin_wait_1'
+              stability: experimental
+            - id: fin_wait_2
+              value: 'fin_wait_2'
+              stability: experimental
+            - id: last_ack
+              value: 'last_ack'
+              stability: experimental
+            - id: listen
+              value: 'listen'
+              stability: experimental
+            - id: syn_recv
```

Comment:
RFC uses `SYN-RECEIVED`

Reply:
Same as https://github.com/open-telemetry/semantic-conventions/pull/1632#discussion_r1867330961. 

It is `SYN_RECV` for Linux while `SYN_RECEIVED` for Windows.

Reply:
replied there, my suggestion is to stick to RFC and it should be easy to translate enum const to otel attribute value string anyway.

Reply:
Tuned the PR according to the rfc9293.